testdata/testinput26

```diff
@@ -11,17 +11,17 @@
 
 # Script extension check
 /^\p{Latin}/utf
-  \x{363}
+  \x{b7}
```

Comment:
Why these characters are changed? Maybe it would be better to keep both.

Reply:
UCD 16 just added that character (which is not even a letter) to a lot of scripts, wonder if we should keep test26 AS-IS and use this for a new test27 instead.

Reply:
Adding a new test like this is a bit more than what I can chew :(

You are right, the test26 diff got unnecessarily big because of these changes.

Reply:
It is actually not that difficult, you would need to hand modify testinput26 so it passes (something like I posted above would work) move the generated files with a `*27` name instead and add that test to `RunTest` and `RunTest.bat`